base/src/main/java/com/mindera/skeletoid/webview/cookies/CookiesUtils.kt

```diff
@@ -34,28 +34,32 @@ object CookiesUtils {
         return cookiesMap
     }
 
-    fun getCookie(url: String, cookieName: String): String? {
+    fun getCookie(url: String, cookieName: String): Map<String,String> {
         var cookieValue: String? = null
 
         val cookieManager = CookieManager.getInstance()
         val cookies = cookieManager.getCookie(url)
 
-        val cookieArray = cookies.split(";").dropLastWhile { it.isEmpty() }.toTypedArray()
+        return stringCookiesToMap(cookies).filterNot { it.key.contains(cookieName) }
```

Comment:
Why are we filter out cookies with that `cookieName`? from the method name I'd assume the contrary.